dmutils/flask.py

```diff
@@ -45,7 +46,14 @@ def __init__(self, *args, compress_by_default: bool = False, **kwargs):
 
     def after_request(self, response):
         x_compression_safe = response.headers.pop("X-Compression-Safe", None)
-        compress = {"0": False, "1": True}.get(x_compression_safe, self.compress_by_default)
+        known_values = {"0": False, "1": True}
+        compress = known_values.get(x_compression_safe, self.compress_by_default)
+
+        if x_compression_safe not in known_values and x_compression_safe != "" and x_compression_safe is not None:
+            warnings.warn(
+                f"{self.__class__.__name__} received unknown X-Compression-Safe header value {x_compression_safe!r} - "
+                "falling back to default"
+            )
```

Comment:
Do we use the built in `warnings` anywhere else? How do these look when fed through an app logger?

Reply:
I don't think they come out at all in logs - I'm expecting people to notice/get annoyed by them being shown after a pytest run.